x.py

```diff
@@ -131,6 +131,36 @@ def cppcheck(args):
 
     run([command, *options, *sources])
 
+def package_source(args):
+    version = args.release_version.strip()
+    if SEMVER_REGEX.match(version) is None:
+        raise RuntimeError(f"Kvrocks version should follow semver spec, got: {version}")
+    
+    # 0. Write input version to VERSION file
+    with open('VERSION', 'w+') as f:
+        f.write(version)
+
+    # 1. Git commit and tag
+    git = find_command('git', msg='git is required for source packaging')
```

Comment:
We don't push anything to remote here. The release manager should manually `git push --tags` to push the tag.

In my opinion the workflow should be:

1. Checkout a staging branch - if we do lightweight release, it's possible to run on `unstable` branch.
2. Run `./x.py package source` to get the source tarball, `.asc` sign file, `.sha512` checksum file.
3. Copy those files to SVN (https://dist.apache.org/repos/dist/dev/incubator/kvrocks).
4. Manually push tags (maybe we can have a `-rcN` tailer, but I don't make it too complex now.
5. Calling a vote.